### PR TITLE
mount nfs from "homes" CNAME instead of from dataloss

### PR DESCRIPTION
**Pull request by cg505**

### CODE REVIEW

**Review by ja5087 — APPROVED** [review]
lgtm

**Review by jvperrin — COMMENTED** [review]
Isn't this the same, why does this make a difference, so the server name isn't hardcoded?

**Comment by cg505** [comment]
yes

**Comment by cg505** [comment]
this is the mount point we use for VMs

File: kubernetes/create-deployment.yml.erb [hunk 1]
```diff
@@ -51,11 +51,11 @@ spec:
         volumes:
           - name: nfs-export-home
             nfs:
-              server: dataloss
+              server: homes
               path: /opt/homes/home
           - name: nfs-export-services
             nfs:
-              server: dataloss
+              server: homes
```
Comment (jvperrin) [inline kubernetes/create-deployment.yml.erb]: May I suggest using `services` here instead? We may split homes and services hosting at some point (although likely not). That being said, the path is still under `/opt/homes` so it's a bit strange.

**Review by jvperrin — APPROVED** [review]
Sure, fewer hardcoded hostnames are always good

**Review by jvperrin — APPROVED** [review]

**Pull request merged**